### ISSUE TITLE
injection issue ig?

### ISSUE THREAD

**Issue by cwelweb** [h1meji/fusion-plus#3]

**Comment by cwelweb** [comment]
I compiled the reproisotry got the dll injected using process hacker and it crashes the game 
last things seen 
09:44:20Exception in thread "Thread-17" java.lang.NoSuchMethodError: Lnet/minecraft/util/BlockPos;.<init>(FFF)V
09:44:26[CHAT] Gohxn France: q ranked skywars beat me = lunar cloak
09:44:40[ LOG ] :: Got Java ENV
09:44:40[ LOG ] :: Attached to Java VM
09:44:40[ LOG ] :: Java initialized
09:44:40[ LOG ] :: Got Minecraft version
09:44:40[ LOG ] :: Lunar Client 1.8.9 detected
09:44:40[ LOG ] :: Stray cache initialized
09:44:40[ LOG ] :: Got Minecraft instance
09:44:40[ LOG ] :: Got Player instance
09:44:40[ LOG ] :: Got World instance
09:44:40[ LOG ] :: Got ActiveRenderInfo instance
09:44:40[ LOG ] :: Got RenderManager instance
09:44:40[ LOG ] :: Got Timer instance
09:44:40[ LOG ] :: Got GameSettings instance
09:44:40[ LOG ] :: Initialized Minecraft
09:44:41[ LOG ] :: Menu initialized
09:44:41[ LOG ] :: Modules initialized
09:44:41[ LOG ] :: Friends loaded
(then game closes)

your provided dll works flawlesly

**Comment by cwelweb** [comment]
```java
			Java::AssignClass("net.minecraft.util.BlockPos", blockPos_class);
			blockPos_constructorInt = Java::Env->GetMethodID(blockPos_class, "<init>", "(III)V");
			blockPos_constructorFloat = Java::Env->GetMethodID(blockPos_class, "<init>", "(FFF)V");
			````

**Comment by cwelweb** [comment]
seems like it it not finding the block pos class

**Comment by cwelweb** [comment]
client crashes uppon conenctiont to server

**Comment by h1meji** [comment]
Thank you for letting me know. When adding support for vanilla 1.8.9, I messed some things up with getting classes and fields.
Fixed in [b018f77](https://github.com/h1meji/fusion-plus/commit/b018f773a0de3fc40423a0e149a35155cd415014).
Let me know if the new version works.

**Comment by cwelweb** [comment]
Fixed, thanks!
